source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -0,0 +1,972 @@
+#Copyright (C) 2014-2015 James Teh, Michael Curren, NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import eventHandler
+import time
+import ui
+from . import Window
+import ctypes
+import math
+import controlTypes
+import colors
+import inputCore
+import re
+from logHandler import log
+
+#This file contains chart constants common to Chart Object in MS Word and MS Excel.
+
+#definitions 
+xlListSeparator = 5
+
+# values for enumeration 'XlChartItem'
+xlDataLabel = 0
+xlChartArea = 2
+xlSeries = 3
+xlChartTitle = 4
+xlWalls = 5
+xlCorners = 6
+xlDataTable = 7
+xlTrendline = 8
+xlErrorBars = 9
+xlXErrorBars = 10
+xlYErrorBars = 11
+xlLegendEntry = 12
+xlLegendKey = 13
+xlShape = 14
+xlMajorGridlines = 15
+xlMinorGridlines = 16
+xlAxisTitle = 17
+xlUpBars = 18
+xlPlotArea = 19
+xlDownBars = 20
+xlAxis = 21
+xlSeriesLines = 22
+xlFloor = 23
+xlLegend = 24
+xlHiLoLines = 25
+xlDropLines = 26
+xlRadarAxisLabels = 27
+xlNothing = 28
+xlLeaderLines = 29
+xlDisplayUnitLabel = 30
+xlPivotChartFieldButton = 31
+xlPivotChartDropZone = 32
+XlChartItem = ctypes.c_int # enum
+
+# Chart types in Microsoft Excel.
+xl3DArea = -4098
+xl3DAreaStacked    = 78
+xl3DAreaStacked100 = 79
+xl3DBarClustered = 60
+xl3DBarStacked = 61
+xl3DBarStacked100 = 62
+xl3DColumn = -4100
+xl3DColumnClustered = 54
+xl3DColumnStacked = 55
+xl3DColumnStacked100 = 56
+xl3DLine = -4101
+xl3DPie = -4102
+xl3DPieExploded = 70
+xlArea = 1
+xlAreaStacked = 76
+xlAreaStacked100 = 77
+xlBarClustered = 57
+xlBarOfPie = 71
+xlBarStacked = 58
+xlBarStacked100 = 59
+xlBubble = 15
+xlBubble3DEffect = 87
+xlColumnClustered = 51
+xlColumnStacked = 52
+xlColumnStacked100 = 53
+xlConeBarClustered = 102
+xlConeBarStacked = 103
+xlConeBarStacked100 = 104
+xlConeCol = 105
+xlConeColClustered = 99
+xlConeColStacked = 100
+xlConeColStacked100 = 101
+xlCylinderBarClustered = 95
+xlCylinderBarStacked = 96
+xlCylinderBarStacked100 = 97
+xlCylinderCol = 98
+xlCylinderColClustered = 92
+xlCylinderColStacked = 93
+xlCylinderColStacked100 = 94
+xlDoughnut = -4120
+xlDoughnutExploded = 80
+xlLine = 4
+xlLineMarkers = 65
+xlLineMarkersStacked = 66
+xlLineMarkersStacked100 = 67
+xlLineStacked = 63
+xlLineStacked100 = 64
+xlPie = 5
+xlPieExploded = 69
+xlPieOfPie = 68
+xlPyramidBarClustered = 109
+xlPyramidBarStacked = 110
+xlPyramidBarStacked100 = 111
+xlPyramidCol = 112
+xlPyramidColClustered = 106
+xlPyramidColStacked = 107
+xlPyramidColStacked100 = 108
+xlRadar = -4151
+xlRadarFilled = 82
+xlRadarMarkers = 81
+xlStockHLC = 88
+xlStockOHLC = 89
+xlStockVHLC = 90
+xlStockVOHLC = 91
+xlSurface = 83
+xlSurfaceTopView = 85
+xlSurfaceTopViewWireframe = 86
+xlSurfaceWireframe = 84
+xlXYScatter = -4169
+xlXYScatterLines = 74
+xlXYScatterLinesNoMarkers = 75
+xlXYScatterSmooth = 72
+xlXYScatterSmoothNoMarkers = 73
+
+# Dictionary for the Description of chart types.
+chartTypeDict = {
+    # Translators: A type of chart in Microsoft Excel. 
```

Comment:
Change this and all other comments to mention Microsoft Office, not Microsoft Excel specifically.